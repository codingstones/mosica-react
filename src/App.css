.App {
  text-align: center;
}

.App-logo {
  height: 50px;
}

.App-header {
  background-color: black;
  height: 100px;
  padding: 5px;
  color: white;
}

.App-title {
  font-size: 1.5em;
}

.App-intro {
  font-size: large;
}

.gigs {
  background-color: #222222;
}

.row {
  background-color: #222222;
  color: #ffa300;
}

.day {
  color: white;
  font-size: x-large;
}
