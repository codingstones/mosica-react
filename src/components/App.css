h2 {
    margin: 1em !important;
}

.content {
    margin: 1em !important;
}

span {
    margin: 1em !important;
}