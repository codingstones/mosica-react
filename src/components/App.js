import React from 'react'
import Main from './Main';
import Header from './Header';

const App = () => (
  <div className="App">
    <Header />
    <Main />
  </div>
)

export default App
