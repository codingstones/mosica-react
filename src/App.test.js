import { shallow, mount } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';
// jest.unmock('mosica-core')

it('renders App', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders App with async', async () => {
  const component = mount(
    <App />
  );
  await flushPromises();
  console.log('.day', component.find('.day'));
  console.log('COMPLETO', component.html());
});

