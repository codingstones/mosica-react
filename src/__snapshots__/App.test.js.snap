// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.png"
    />
    <h1
      className="App-title"
    >
      Mosica React
    </h1>
  </header>
  <div
    className="Day"
  >
    DAY A BORRAR1
  </div>
</div>
`;
